byzcoin/api.go

```diff
@@ -212,6 +229,37 @@ func (c *Client) GetProofFrom(key []byte, from *skipchain.SkipBlock) (*GetProofR
 	return reply, nil
 }
 
+// GetDeferredData makes a request to retrieve the deferred instruction data
+// and return the reply if the proof can be verified.
+func (c *Client) GetDeferredData(instrID InstanceID) (*DeferredData, error) {
```

Comment:
Is this correctly included here, or should this be a separate PR?

Reply:
I was changing the client transaction when I saw a lot of code repetition so I made this function. There's no new feature from it. If that was the question.